{
  "className": "qx.core.AssertionError",
  "packageName": "qx.core",
  "name": "AssertionError",
  "superClass": "qx.type.BaseError",
  "interfaces": [],
  "mixins": [],
  "functionName": null,
  "clazz": {
    "location": {
      "start": {
        "line": 23,
        "column": 0
      },
      "end": {
        "line": 68,
        "column": 3
      }
    },
    "jsdoc": {
      "@description": [
        {
          "name": "@description",
          "body": "***********************************************************************\n\nqooxdoo - the new era of web development\n\nhttp:\n\nCopyright:\n2007-2008 1&1 Internet AG, Germany, http:\n\nLicense:\nMIT: https:\nSee the LICENSE file in the project's top-level directory for details.\n\nAuthors:\nFabian Jakobs (fjakobs)\n\n***********************************************************************"
        },
        {
          "name": "@description",
          "body": "Assertion errors are thrown if an assertion in {@link qx.core.Assert}\nfails."
        }
      ]
    }
  },
  "type": "class",
  "construct": {
    "location": {
      "start": {
        "line": 40,
        "column": 14
      },
      "end": {
        "line": 44,
        "column": 3
      }
    }
  },
  "members": {
    "__trace": {
      "location": {
        "start": {
          "line": 56,
          "column": 4
        },
        "end": {
          "line": 56,
          "column": 18
        }
      },
      "type": "variable",
      "access": "private"
    },
    "getStackTrace": {
      "location": {
        "start": {
          "line": 64,
          "column": 4
        },
        "end": {
          "line": 66,
          "column": 5
        }
      },
      "jsdoc": {
        "@description": [
          {
            "name": "@description",
            "body": "Stack trace of the error"
          }
        ],
        "@return": [
          {
            "name": "@return",
            "body": "{String[]} The stack trace of the location the exception was thrown",
            "type": "String[]",
            "description": " The stack trace of the location the exception was thrown"
          }
        ]
      },
      "type": "function",
      "access": "public"
    }
  },
  "descendants": []
}